OpenRA.Mods.Common/Traits/Buildings/Gate.cs

```diff
@@ -0,0 +1,144 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public interface ITemporaryBlocker
```

Comment:
Not all of our interfaces are there.
For example `IPreventsEjectOnDeath`.

It just depends on if it is specific to a trait, do we have another use for this currently?